src/main/java/io/vertx/blueprint/todolist/service/TodoService.java

```diff
@@ -0,0 +1,31 @@
+package io.vertx.blueprint.todolist.service;
+
+import io.vertx.blueprint.todolist.entity.Todo;
+import io.vertx.core.Future;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Vert.x Blueprint Application - Todo Backend
+ * Todo Service Interface
+ *
+ * @author Eric Zhao
+ */
+public interface TodoService {
+
+  Future<Boolean> initData();
```

Comment:
The `initData` method is intended to init the persistence(e.g create table if not exists) as well as check the connection. (I thought it needs to be invoked only during the init of the verticle?)